### PR TITLE
Adding navigation buttons

### PR DESCRIPTION
**Pull request by gvn**
This is a start on adding the directional button navigation for Play mode.

It still needs more styling, but functionally it seems to work. It also needs to be integrated into `webmaker-browser`, but I'm trying to make minimal changes to `project.jsx` so that shouldn't be too bad.

### CODE REVIEW

File: src/pages/project/project.jsx [hunk 1]
```diff
@@ -86,6 +96,12 @@ var Project = React.createClass({
     var isPlayOnly = (mode === 'play' || mode === 'link');
     return (
       <div id="map" className={this.state.params.mode}>
+        <DPad
```
Comment (k88hudson) [inline src/pages/project/project.jsx:99]: haha love the name

**Comment by k88hudson** [comment]
Awesome! Can you file a follow-up ticket to sync up with design to refine the UI a bit? You can just do the `webmaker-browser` patch as part of this ticket though if you want

R+ otherwise

File: src/pages/project/project.jsx (hunk 1 above)
Comment (gvn) [inline src/pages/project/project.jsx:99]: :space_invader:

**Comment by gvn** [comment]
Yep, for sure. I'm leaving #292 open w. a checklist...

**Pull request merged**